


{
    "language.name": "English",
    "language.region": "United States",
    "language.code": "en_us",


    "commands.permissions.insufficient" : "You dont have enough permissions",
    "commands.permissions.insufficient.manage" : "You dont have enough permissions to manage {Player}",
    "commands.permissions.insufficient.addrole" : "You dont have enough permissions to manage the role {Rolename}",

    "commands.dm.cant_dm_yourself" : "You cant dm yourself",
    "commands.dm.missing_args" : "Usage : {Prefix}dm <targetname> <message>",
    "commands.dm.from" : "{Player} - me : {Message}",
    "commands.dm.to" : "me - {Player} : {Message}",
    "commands.not_found" : "Command {Command} not found",
    "player.not_found" : "Player {Player} not found",

    "database.code.nickel.nochange" : "Nothing changed",

    "database.code.0" : "Successfully executed",
    "database.code.1" : "SQL error or missing database",
    "database.code.2" : "Internal logic error in SQLite",
    "database.code.3" : "Server file access denied (Server doesn't have enough permissions)",
    "database.code.4" : "Callback routine requested an abort",
    "database.code.5" : "The database file is locked",
    "database.code.6" : "A table in the database is locked",
    "database.code.7" : "A malloc() failed",
    "database.code.8" : "Attempt to write a readonly database",
    "database.code.9" : "Operation terminated by sqlite3_interrupt()",
    "database.code.10" : "Some kind of disk I/O error occurred",
    "database.code.11" : "The database disk image is malformed",
    "database.code.12" : "(Internal Only) Table or record not found",
    "database.code.13" : "Insertion failed because the database is full",
    "database.code.14" : "Unable to open the database file",
    "database.code.15" : "Database lock protocol error",
    "database.code.16" : "Database is empty",
    "database.code.17" : "The database schema changed",
    "database.code.18" : "String or BLOB exceeds size limit",
    "database.code.19" : "Abort due to constraint violation. This entry probably already exists",
    "database.code.20" : "Data type mismatch",
    "database.code.21" : "Library used incorrectly",
    "database.code.22" : "Uses OS features not supported on the host",
    "database.code.23" : "Authorization denied",
    "database.code.24" : "Auxiliary database format error",
    "database.code.25" : "2nd parameter to sqlite3_bind out of range",
    "database.code.26" : "File opened that is not a database file",
    "database.code.27" : "Notifications from sqlite3_log()",
    "database.code.28" : "Warnings from sqlite3_log()",




    "database.code.516" : "Abort due to constraint violation",
    "database.code.279" : "Authorization denied",
    "database.code.261" : "Database file is locked during recovery",
    "database.code.517" : "Database is locked due to a recovery operation",
    "database.code.773" : "Database lock timeout",
    "database.code.1038" : "Unable to open the database file with the requested encoding",
    "database.code.1294" : "Database file is read-only due to a rollback journal",
    "database.code.782" : "Unable to open the database file with the requested full path",
    "database.code.526" : "The file is a directory",
    "database.code.270" : "No such file or directory",
    "database.code.1550" : "Unable to open the database file due to a symbolic link",
    "database.code.275" : "CHECK constraint failed",
    "database.code.531" : "CHECK constraint failed during the execution of a COMMIT",
    "database.code.3091" : "Datatype mismatch",
    "database.code.787" : "FOREIGN KEY constraint failed",
    "database.code.1043" : "User-defined function called with an incorrect argument",
    "database.code.1299" : "NOT NULL constraint failed",
    "database.code.2835" : "NOT NULL constraint failed due to pinned shared cache",
    "database.code.1555" : "PRIMARY KEY constraint failed",
    "database.code.2579" : "ROWID constraint failed",
    "database.code.1811" : "Trigger program or foreign key action program failed",
    "database.code.2067" : "UNIQUE constraint failed",
    "database.code.2323" : "VIRTUAL TABLE constraint failed",
    "database.code.779" : "Corrupt index",
    "database.code.523" : "Corrupt sequence",
    "database.code.267" : "Corrupt virtual table",
    "database.code.257" : "Internal schema object missing",
    "database.code.513" : "Retry the operation",
    "database.code.769" : "Operation was blocked due to another process recovering a WAL mode database",
    "database.code.3338" : "Access permission denied during an attempt to access the database file",
    "database.code.7178" : "Authorization denied during an attempt to access the database file",
    "database.code.7434" : "A database file is locked during an atomic commit",
    "database.code.2826" : "Database file is locked due to another process",
    "database.code.3594" : "Database file is locked when trying to execute a CHECKRESERVEDLOCK operation",
    "database.code.4106" : "Database file is locked during an attempt to close the file",
    "database.code.7690" : "Database file is locked during an atomic commit",
    "database.code.6666" : "Unable to open the database file with the requested encoding",
    "database.code.8458" : "Database file is corrupt",
    "database.code.8202" : "Read or write operation failed because of an I/O error",
    "database.code.2570" : "File delete operation failed",
    "database.code.5898" : "File delete operation failed because the file does not exist",
    "database.code.4362" : "Error when trying to close a directory",
    "database.code.1290" : "Error when trying to sync the directory",
    "database.code.1802" : "Error when trying to fstat a file",
    "database.code.1034" : "Error when trying to fsync a file",
    "database.code.6410" : "Error when trying to get the temporary directory",
    "database.code.3850" : "Error when trying to lock the file",
    "database.code.6154" : "Error when trying to mmap a file",
    "database.code.3082" : "Out of memory",
    "database.code.2314" : "Error when trying to acquire a READ lock on the file",
    "database.code.266" : "Error when trying to read from a file",
    "database.code.7946" : "Error during an atomic rollback",
    "database.code.5642" : "Error when trying to seek to a specific offset in a file",

    "database.code.5130" : "Error when trying to lock the shared memory region",
    "database.code.5386" : "Error when trying to map the shared memory region",
    "database.code.4618" : "Error when trying to open the shared memory region",
    "database.code.4874" : "Error when trying to set the size of the shared memory region",
    "database.code.522" : "Error when trying to read a short number of bytes from a file",
    "database.code.1546" : "Error when trying to truncate a file",
    "database.code.2058" : "Error when trying to unlock a file"
    


}